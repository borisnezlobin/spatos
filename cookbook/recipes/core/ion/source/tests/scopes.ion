if test 1 == 1
  let x = 5

  if test 1 == 1
    let x = 2
    let y = 3
  end

  echo $x
  echo $y
  echo Should not be printed
end

