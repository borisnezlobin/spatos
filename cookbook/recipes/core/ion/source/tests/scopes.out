2
ion: expansion error: Variable "y" does not exist
